Packs/DomainTools/Integrations/DomainTools/DomainTools.js

```diff
@@ -300,27 +300,31 @@ var callWhoisHistory = function(url, domain){
 }
 
 var url = params.server.replace(/[\/]+$/, '');
+var key = (params.key || params.credentials.password);
```

Comment:
```suggestion
params.key = params.key || params.credentials.password
```
then you don't need to add key as an argument everywhere

Reply:
done